quickstart/examples/inference-scheduling/ms-inference-scheduling/values_tpu.yaml

```diff
@@ -0,0 +1,123 @@
+# This values.yaml file creates the resources for Qwen/Qwen3-0.6B
```

Comment:
What's the diff between this and the GPU example? Is it just the image and the node selector for TPU? If so, instead of creating a new values file, we can just override for TPU like so 

```
helmfile apply --set decode.containers.image="tpu_image" 
```

Reply:
looks like there are more diffs like resource, so using a tpu.yaml looks good